source/install.sql

```diff
@@ -36,20 +70,53 @@ whenever oserror exit failure rollback
 @@types/ut_reporter_decorator.tpb
 @@types/ut_dbms_output_suite_reporter.tpb
 @@ut_metadata.pkb
-@@ut_annotations.pkb
+@@expectation_data_values/ut_data_value_anydata.tpb
+@@expectation_data_values/ut_data_value_blob.tpb
+@@expectation_data_values/ut_data_value_boolean.tpb
+@@expectation_data_values/ut_data_value_clob.tpb
+@@expectation_data_values/ut_data_value_date.tpb
+@@expectation_data_values/ut_data_value_number.tpb
+@@expectation_data_values/ut_data_value_refcursor.tpb
+@@expectation_data_values/ut_data_value_timestamp.tpb
+@@expectation_data_values/ut_data_value_timestamp_tz.tpb
+@@expectation_data_values/ut_data_value_timestamp_ltz.tpb
+@@expectation_data_values/ut_data_value_varchar2.tpb
+@@expectations/ut_expectation.tpb
+@@expectations/be_false.tpb
+@@expectations/be_like.tpb
+@@expectations/be_not_null.tpb
+@@expectations/be_null.tpb
+@@expectations/be_true.tpb
+@@expectations/equal.tpb
+@@expectations/match.tpb
+@@assertions/ut_assert_processor.pkb
+@@assertions/ut_assertion.tpb
+@@assertions/ut_assertion_anydata.tpb
+@@assertions/ut_assertion_blob.tpb
+@@assertions/ut_assertion_boolean.tpb
+@@assertions/ut_assertion_clob.tpb
+@@assertions/ut_assertion_date.tpb
+@@assertions/ut_assertion_number.tpb
+@@assertions/ut_assertion_refcursor.tpb
+@@assertions/ut_assertion_timestamp.tpb
+@@assertions/ut_assertion_timestamp_ltz.tpb
+@@assertions/ut_assertion_timestamp_tz.tpb
+@@assertions/ut_assertion_varchar2.tpb
+@@assertions/ut.pkb
 @@ut_assert.pkb
+@@ut_annotations.pkb
 @@ut_suite_manager.pkb
 
 
 prompt Validating installation
-select * from user_errors where name not like 'BIN$%' and name like 'UT\_%' escape '\';
+select * from user_errors where name not like 'BIN$%' and (name like 'UT%' or name in ('EQUAL','BE_TRUE','BE_FALSE','BE_NULL','BE_NOT_NULL','MATCH','BE_LIKE'));
```

Comment:
in fact.. can you add `and attribute = 'ERROR'` also..  as by default i have WARNINGS turned on and it "fails to install" becuase of warnings